migration/template/upgrade-apps.md

```diff
@@ -0,0 +1,55 @@
+# Upgrade v${old_version}.x to v${new_version}.x
+
+## Prerequisites
+
+- [ ] Check if there are any pending changes to the environment;
+- [ ] Check the state of the environment, pods, nodes and backup jobs:
+
+    ```bash
+    ./bin/ck8s test sc|wc
+    kubectl get po -A
+    kubectl get nodes
+    kubectl get jobs -A
+    velero get backup
+    ```
+
+- [ ] Enable maintenance mode for the monitoring solution;
+- [ ] Notify the on-call engineers and the users before the maintenance starts.
```

Comment:
I would say that 
- Check if there are any pending changes to the environment;
- Enable maintenance mode for the monitoring solution;
- Notify the on-call engineers and the users before the maintenance starts.
is too specific.

Reply:
why do you think that?
I see them quite general and applicable for every application maintenance, not only compliantkubernetes-apps

Reply:
Well, 
- `Check if there are any pending changes to the environment;` more or less assumes our internal workflow with github PRs. This is definitely not a general case.
- `Enable maintenance mode for the monitoring solution;`, not every user as an external monitoring solution.
- `Notify the on-call engineers and the users before the maintenance starts.`, is very specific to us.

Reply:
`Check if there are any pending changes to the environment` - yes it's assumes that the overwrite configs are keep in a version control system. in my view a common use case
`Enable maintenance mode for the monitoring solution` - true, but also alertmanager has also a similar function ([silences](https://prometheus.io/docs/alerting/latest/alertmanager/#silences)), I can change the wording to also highlight that
`Notify the on-call engineers and the users before the maintenance starts` - agree I can replace maintenance with upgrade and also drop the on-call engineers. regarding users, as they are supported for wc and mentioned in the docs I want to keep them